packages/esm-active-visits-app/src/active-visits-widget/active-visits.component.tsx

```diff
@@ -168,116 +188,114 @@ const ActiveVisitsTable = () => {
-                        {row.cells.map((cell) => (
-                          <TableCell key={cell.id} data-testid={cell.id}>
-                            {cell.info.header === 'name' ? (
-                              <PatientNameLink
-                                from={fromPage}
-                                to={`\${openmrsSpaBase}/patient/${paginatedActiveVisits?.[index]?.patientUuid}/chart/`}>
-                                {cell.value}
-                              </PatientNameLink>
-                            ) : (
-                              cell.value
-                            )}
-                          </TableCell>
-                        ))}
-                      </TableExpandRow>
-                      {row.isExpanded ? (
-                        <TableRow className={styles.expandedActiveVisitRow}>
-                          <th colSpan={headers.length + 2}>
-                            <ExtensionSlot
-                              className={styles.visitSummaryContainer}
-                              extensionSlotName="visit-summary-slot"
-                              state={{
-                                visitUuid: paginatedActiveVisits[index]?.visitUuid,
-                                patientUuid: paginatedActiveVisits[index]?.patientUuid,
-                              }}
-                            />
-                          </th>
-                        </TableRow>
-                      ) : (
-                        <TableExpandedRow className={styles.hiddenRow} colSpan={headers.length + 2} />
-                      )}
-                    </React.Fragment>
-                  ))}
-                </TableBody>
-              </Table>
-              {rows.length === 0 && (
-                <p
-                  style={{
-                    height: isDesktop(layout) ? '2rem' : '3rem',
-                    margin: '1rem 1.5rem',
-                  }}
-                  className={`${styles.emptyRow} ${styles.bodyLong01}`}>
-                  {t('noVisitsFound', 'No matching visits found')}
-                </p>
-              )}
-              <Pagination
-                forwardText="Next page"
-                backwardText="Previous page"
-                page={currentPage}
-                pageSize={currentPageSize}
-                pageSizes={pageSizes}
-                totalItems={searchResults.length}
-                className={styles.pagination}
-                onChange={({ pageSize, page }) => {
-                  if (pageSize !== currentPageSize) {
-                    setPageSize(pageSize);
-                  }
-                  if (page !== currentPage) {
-                    goTo(page);
-                  }
-                }}
-              />
-            </TableContainer>
-          )}
-        </DataTable>
       </div>
-    );
-  }
+      <Search
+        // tabIndex={getBatchActionProps().shouldShowBatchActions ? -1 : 0}
```

Comment:
could we just remove this comment to clean up the code?

Reply:
Sure!
Thanks!